workflows/metatdenovo.nf

```diff
@@ -73,17 +73,21 @@ include { PROKKA_CAT } from '../subworkflows/local/prokka_cat'
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
+//subread_featurecounts_options_cds        = modules['subread_featurecounts_cds']
+
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC                      } from '../modules/nf-core/modules/fastqc/main'
-include { BBMAP_BBDUK                 } from '../modules/nf-core/modules/bbmap/bbduk/main'
-include { BBMAP_INDEX                 } from '../modules/nf-core/modules/bbmap/index/main'
-include { BBMAP_ALIGN                 } from '../modules/nf-core/modules/bbmap/align/main'
-include { SEQTK_MERGEPE               } from '../modules/nf-core/modules/seqtk/mergepe/main'
-include { PRODIGAL                    } from '../modules/nf-core/modules/prodigal/main'
-include { MULTIQC                     } from '../modules/nf-core/modules/multiqc/main'
-include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main'
+include { FASTQC                                 } from '../modules/nf-core/modules/fastqc/main'
+include { BBMAP_BBDUK                            } from '../modules/nf-core/modules/bbmap/bbduk/main'
+include { BBMAP_INDEX                            } from '../modules/nf-core/modules/bbmap/index/main'
+include { BBMAP_ALIGN                            } from '../modules/nf-core/modules/bbmap/align/main'
+include { SEQTK_MERGEPE                          } from '../modules/nf-core/modules/seqtk/mergepe/main'
+include { BAM_SORT_SAMTOOLS                      } from '../subworkflows/nf-core/bam_sort_samtools/main'
+include { SUBREAD_FEATURECOUNTS as FEATURECOUNTS } from '../modules/nf-core/modules/subread/featurecounts/main'
```

Comment:
Change `FEATURECOUNTS` to `FEATURECOUNTS_CDS` here too. You'll have to move all those right curly brackets again!